### DISCUSSION TITLE
Hide source code when deploy

### DISCUSSION THREAD

**Discussion by adrianzy** [fastapi/fastapi#3347] [Questions]
Is it possible to deploy a complied source code?
How to use with Pyinstaller?

**Comment by acidjunk** [comment]
Python code can be decompiled quite easy; if you know the python version. (which can be found in the beginning of the compiled code) if I am not mistaken.

**Comment by Atem18** [comment]
@adrianzy You can use my tutorial if you want: https://www.atemlire.io/how-to-create-a-small-and-secure-container-for-your-python-applications/
